tests/cases/providers_care_platform/case_providers_home.yaml

```diff
@@ -497,4 +497,102 @@ TestProviderHelpMenuOption:
           Var: HELP_PAGE_URL
           Value: .pdf
 
- 
\ No newline at end of file
+# CXTB-185 Provider Can Click and View Facility Device Record from Listing
+TestProviderCanClickAndViewFacilitiesDevice:
+  Environment: Settings
+  Roles:
+    - Role: desktopChrome
+      App: "desktop"
+  Actions:
+    # Login-in provider Care Partner
+    - Type: case
+      Value: ProvidersCarePlatformLogin
+    # Click facilities button
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_facility.facilities_button$
+    # click on the first listing record
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant.first_senior_lastname$
```

Comment:
This should be a facility record not a participant last name

Reply:
fixed, please check